examples/multiple-choice/run_swag.py

```diff
@@ -312,7 +312,12 @@ def preprocess_function(examples):
 
     # Data collator
     data_collator = (
-        default_data_collator if data_args.pad_to_max_length else DataCollatorForMultipleChoice(tokenizer=tokenizer)
+        default_data_collator
+        if data_args.pad_to_max_length
+        else DataCollatorForMultipleChoice(
+            tokenizer=tokenizer,
+            pad_to_multiple_of=8 if training_args.fp16 else None,
```

Comment:
```suggestion
            tokenizer=tokenizer, pad_to_multiple_of=8 if training_args.fp16 else None,
```
I think this can fit in one line, perhaps even the same line as before.

Reply:
> Thanks for your PR, that's great! One last thing is to adapt the template example [here](https://github.com/huggingface/transformers/blob/master/templates/adding_a_new_example_script/%7B%7Bcookiecutter.directory_name%7D%7D/run_%7B%7Bcookiecutter.example_shortcut%7D%7D.py) if you don't mind.

sure, I missed that. Let me know if this works.